libopae/plugins/xfpga/metrics/metrics_utils.c

```diff
@@ -328,6 +331,9 @@ fpga_result enum_powermgmt_metrics(fpga_metric_vector *vector,
 
 	for (i = 0; i < pglob.gl_pathc; i++) {
 
+		if (!pglob.gl_pathv)
```

Comment:
same thing.. should this emit a warning at least?

Reply:
fixed